### ISSUE TITLE
Popover Dialog DismissButton

### ISSUE THREAD

**Issue by ashleyryan** [adobe/react-spectrum#8159]
### Provide a general summary of the issue here

The Dismiss Button cannot be accessed in the popover on the `usePopover` examples: https://react-spectrum.adobe.com/react-aria/usePopover.html#usepopover

### 🤔 Expected Behavior?

I should be able to tab when in voiceover to reach the dismiss button in a popover to close it

### 😯 Current Behavior

When I use voiceover (or even when I'm not using voiceover) I can't seem to reach the dismiss button to close the popover/dialog

### 🔦 Context

We're currently undergoing an accessibility audit and had used `usePopover` without the `<Dialog>` inside it for non-interactive popovers. We set role=tooltip on them. The auditors felt that our popovers contained too much information to be considered a tooltip and were actually a toggletip, which would require a live region and role="status".

We couldn't figure out how to implement a live region when usePopover used a React portal to render the contents, so we asked if we could use role="dialog" all the time instead, and they said yes, however, dialogs must contain a close button. I noticed on your doc site that you have hidden close buttons for accessibility, but I cannot figure out how to reach them via the keyboard or in voiceover because they have tabindex=-1.

I wasn't sure if this was a bug or an issue with how I'm trying to reach the buttons

### 🖥️ Steps to Reproduce

Try to access to the close button inside of the popopvers on the `usePopover` doc site examples

https://react-spectrum.adobe.com/react-aria/usePopover.html#usepopover

### Version

the latest

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

MacOS Sequoia

**Comment by ashleyryan** [comment]
Ok thank you. I'm not sure what's different about my implementation, but it looks like VO does announce control+option+space to close, which the usePopover docs does not (though it works).

FWIW - and I'm sure you have your own A11Y experts at Adobe, but I also just received clarification from our auditors that our popovers should have a visible close button as well for users with cognitive issues. I'm not sure if that's a strict WCAG requirement, but figured I'd share the feedback we got on it.

**Comment by devongovett** [comment]
Yeah I think that's a bit of a gray area that depends on your design. Sometimes it's obvious enough that you can click outside or click the trigger button again to close it. Other times maybe not.

Gonna close this as working as intended for now. If you have more questions let us know!
